build/parents/build/pom.xml

```diff
@@ -0,0 +1,1522 @@
+                        </systemPropertyVariables>
+                    </configuration>
+                </plugin>
+                <!--
+                    Code formatting configuration:
+                -->
+                <plugin>
+                    <groupId>net.revelc.code</groupId>
+                    <artifactId>impsort-maven-plugin</artifactId>
+                    <version>${version.impsort-maven-plugin}</version>
+                    <configuration>
+                        <!-- store outside of target to speed up formatting when mvn clean is used -->
+                        <cachedir>.cache/impsort-maven-plugin-${version.impsort-maven-plugin}</cachedir>
+                        <groups>java.,javax.,jakarta.,org.hibernate., org.hibernate.testing., org.hibernate.test., org.junit., org.jboss., com., *</groups>
+                        <staticGroups>*</staticGroups>
+                        <!-- To make static imports go on top: -->
+                        <staticAfter>false</staticAfter>
+                        <skip>${format.skip}</skip>
+                        <removeUnused>true</removeUnused>
+                    </configuration>
+                    <executions>
+                        <execution>
+                            <id>import-sorting</id>
+                            <goals>
+                                <goal>${goal.impsort-maven-plugin}</goal>
+                            </goals>
+                            <phase>process-sources</phase>
+                        </execution>
+                    </executions>
+                </plugin>
+                <plugin>
+                    <groupId>net.revelc.code.formatter</groupId>
+                    <artifactId>formatter-maven-plugin</artifactId>
+                    <version>${version.formatter-maven-plugin}</version>
+                    <dependencies>
+                        <dependency>
+                            <groupId>org.hibernate.search</groupId>
+                            <artifactId>hibernate-search-build-config</artifactId>
+                            <version>${project.version}</version>
+                        </dependency>
+                    </dependencies>
+                    <configuration>
+                        <!-- store outside of target to speed up formatting when mvn clean is used -->
+                        <cachedir>.cache/formatter-maven-plugin-${version.formatter-maven-plugin}</cachedir>
+                        <configFile>hibernate_search_style.xml</configFile>
+                        <lineEnding>LF</lineEnding>
+                        <skip>${format.skip}</skip>
+                        <removeTrailingWhitespace>true</removeTrailingWhitespace>
+                    </configuration>
+                </plugin>
```

Comment:
This is already in the root POM, the only difference AFAICT being `<skip>${format.skip}</skip>`. Might as well move that to the root POM as well, and remove this entirely?